### ISSUE TITLE
database.json中的grid_tpxo10atlas_v2的路径似乎是出错了，应该改为grid_tpxo10_atlas_30_v2

### ISSUE THREAD

**Issue by eceguhraterfa** [tsutterley/pyTMD#358]

**Comment by robbibt** [comment]
Hey @eceguhraterfa, I think it looks correct for the NetCDF variants of TPXO10 at least - this is the file name in the official OSU TPXO repository:

![image](https://github.com/user-attachments/assets/5a4422b3-0357-4fef-9c10-2b3855995f0e)

**Comment by eceguhraterfa** [comment]
Indeed, the nc version is pictured above, but the bin version is grid_tpxo10_atlas_30_v2.  The nc version's file naming is inconsistent with all the previous ones, although not a fatal problem
![Snipaste_2024-10-31_19-12-09](https://github.com/user-attachments/assets/09bc4f67-ac5c-4f24-865d-5bf7aad8f3e4)

**Comment by tsutterley** [comment]
I'll fix it today!
